src/Application/Application.php

```diff
@@ -267,4 +222,94 @@ private function showScriptNotFoundListing(ScriptNotFoundException $ex, array $s
             $this->showListing($scripts);
         }
     }
+
+    private function printHead(Config $config): void
```

Comment:
What's the difference between `head` and `header`?

Reply:
A Problem in naming. The header is a configured flash - like the enterprise in our project. The head is the output of this in combination with config files.... I'll think of something....

Reply:
I merged the methods